build.sbt

```diff
@@ -321,10 +325,8 @@ lazy val withCssLoading: Project => Project =
     )
   )
 
-/**
-  * Implement the `start` and `dist` tasks defined above.
-  * Most of this is really just to copy the index.html file around.
-  */
+addCommandAlias("dev", ";material-ui/fastOptJS::startWebpackDevServer;~material-ui/fastOptJS")
```

Comment:
call it `muiDev` or something, there are many other demos and this will be global

Reply:
Yes, indeed, I'd propagate the hot reloading to all the demos, and specify the command `dev` command for each of those, I'll update the PR once I get some free time